leader-election-impl/src/main/java/com/palantir/leader/proxy/AwaitingLeadershipProxy.java

```diff
@@ -103,6 +110,7 @@ public void run() {
 
     private void gainLeadership() {
         try {
+            leaderLog.info("Quiescing");
```

Comment:
You're probably right. I feel like "Waiting to become the leader" implies to someone unfamiliar with the code that we've tried to trigger an election, though. Maybe just "Not the leader; going to sleep"?